package.json

```diff
@@ -47,16 +47,21 @@
     "@ngrx/store-devtools": "^6.0.3",
     "@nguniversal/express-engine": "^7.1.0",
     "@types/crypto-js": "^3.1.43",
+    "@types/dotenv": "^6.1.1",
+    "@types/getenv": "^0.7.0",
+    "@types/global-tunnel-ng": "^2.1.0",
```

Comment:
All @types packages should be devDependencies instead of actual dependencies, as they're not needed for the final build.